solutions/13.js

```diff
@@ -13,6 +12,22 @@ const solution = (str) => {
 	return true;
 };
 
+// Maricris Bonzo: @seemcat
+const solution1 = (string) => {
+  let indexLength = string.length - 1;
+  let reverseString = '';
+
+  for(i = 0; i <= indexLength; i++){
+    reverseString = string[i] + reverseString;
```

Comment:
It passes! 
![screen shot 2017-06-15 at 11 21 41 am](https://user-images.githubusercontent.com/11692241/27195759-da1ea006-51bc-11e7-9b7e-4366a661cfd5.png)